contracts/root/predicates/ERC20Predicate.sol

```diff
@@ -144,6 +144,30 @@ contract ERC20Predicate is IErcPredicate {
     return ageOfChallengeTx > age;
   }
 
+  function interpretStateUpdate(bytes calldata state)
+    external
+    view
+    returns(bytes memory)
+  {
+    (bytes memory _data, address participant) = abi.decode(state, (bytes, address));
+    // uint256 age = withdrawManager.verifyInclusion(_data, 0, false /* verifyTxInclusion */);
+    uint256 age;
+    RLPReader.RLPItem[] memory referenceTx = _data.toRlpItem().toList();
+    bytes memory receipt = referenceTx[6].toBytes();
+    uint256 logIndex = referenceTx[9].toUint();
+    require(logIndex < MAX_LOGS, "Supporting a max of 10 logs");
+    RLPReader.RLPItem[] memory inputItems = receipt.toRlpItem().toList();
+    inputItems = inputItems[3].toList()[logIndex].toList(); // select log based on given logIndex
+    ReferenceTxData memory data;
+    data.childToken = RLPReader.toAddress(inputItems[0]); // "address" (contract address that emitted the log) field in the receipt
+    bytes memory logData = inputItems[2].toBytes();
+    inputItems = inputItems[1].toList(); // topics
+    data.rootToken = address(RLPReader.toUint(inputItems[1]));
+    (data.closingBalance, data.age) = processStateUpdate(inputItems, logData, participant);
+    data.age = data.age + age + (logIndex * MAX_LOGS); // @todo use safeMath
```

Comment:
no use of age here ?

Reply:
Was waiting on the parity fix, checkout it now